www/js/scripts-library.js

```diff
@@ -1405,6 +1405,12 @@ $('#context-menu-playback a').click(function(e) {
             $('#setfav-modal').modal();
         });
 	}
+  else if ($(this).data('cmd') == 'song-lyrics') {
```

Comment:
This handler cause a double load on my system, because the one from playerlib.js also triggers ?